_pkgdown.yml

```diff
@@ -18,10 +18,7 @@ navbar:
     introduction:
       text: Introduction
       href: articles/datatable-intro.html
-    reference:
```

Comment:
This tab is still there, added automatically by pkgdown. and when you click on it the hover stays on Vignettes, which is confusing. Will move it back again.